maintainers/maintainer-list.nix

```
@@ -283,6 +283,11 @@
     githubId = 273837;
     name = "Mateusz Czapliński";
   };
+  akamaus = {
+    email = "dmitryvyal@gmail.com";
+    github = "akamaus";
```

```suggestion
    github = "akamaus";
    githubId = 58955;
```
https://gist.github.com/GrahamcOfBorg/e5c60cefeb6835923d8e2041bb22fa0a